myapp/spec/system/tasks_spec.rb

```diff
@@ -54,4 +54,130 @@
       end
     end
   end
+
+  describe '登録画面のバリデーションの動作を確認する', type: :system do
+    context '入力項目を空にして登録ボタンをクリック' do
+      it '未入力の項目のバリデーションエラー文言が表示される' do
+        visit new_task_path
+        click_button 'commit'
+        page.driver.browser.switch_to.alert.accept
```

Comment:
`page.accept_confirm`を使うのが良さそうです！
https://blog.regonn.tokyo/rails/2017-06-13-rspec-capybara/

こんな感じかと思います。
```suggestion
        page.accept_confirm do
          click_button 'commit'
        end
```